tests/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+# build and add test only if tests are enabled
```

Comment:
Just a comment on tests, these are our life jackets, so i think that they should be enabled and executed always by default. Otherwise we never know when we introduce an error in our code (like i did in my las pr)

Reply:
I agree we should enable them on all of our CI builds, but we should not force them on people who just "quickly" want to compile AppImageKit for their own use (e.g., as part of their own app build pipeline) with minimal dependencies.

Reply:
They're only built, but not force-run. That's CMake's default way of handling things.

@azubieta technical detail: `BUILD_TESTING` is set to `ON` by default. Hence, the tests should be built by default.

Reply:
:+1: